static/js/flow/applicationTypes.js

```diff
@@ -64,14 +64,19 @@ export type ApplicationDetail = {
 }
 
 export type SubmissionReview = {
-  id:                      number,
+  id: number,
   run_application_step_id: number,
-  submitted_date:          ?string,
-  review_status:           ?string,
-  review_status_date:      ?string,
-  bootcamp_application:    Application,
-  learner:                 User,
-  interview_url:           ?string
+  submitted_date: ?string,
+  review_status: ?string,
+  review_status_date: ?string,
+  bootcamp_application: Application,
+  learner: User,
+  interview_url: ?string
+}
+
+export type ApplicationResumeDetail = {
+  resume_file:       ?Object,
+  linkedin_url:          ?string,
```

Comment:
Nit: can we go with either single space after the colon, or line up the values? It's neither one here